src/NorthwindWebsite/Presentation/Views/Shared/Components/Breadcrumbs/Breadcrumbs.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace NorthwindWebsite.Presentation.Views.Shared.Components.Breadcrumps;
+
+public class Breadcrumbs : ViewComponent
+{
+    private const string NavigationStartPoint = "Home";
+    private const string HandleActionName = "Handle";
+    private const string ImageUploadActionName = "ImageUpload";
+
+    public IViewComponentResult Invoke()
+    {
+        var path = HttpContext.Request.Path.ToString();
+
+        var processedArray = ProcessSpecialCases(path);
+
+        var readyBreadcrumbs = string.Join(" > ", processedArray);
+
+        var breadcrembs = string.Concat(NavigationStartPoint, readyBreadcrumbs);
```

Comment:
1. I guess you may combine previous and this row:
```suggestion
        var breadcrembs = string
                               .Join(" > ", processedArray)
                               .Concat(NavigationStartPoint, readyBreadcrumbs);
```
3. breadcrumbs*

Reply:
Can't see how this chain is possible, did it like this: NavigationStartPoint + string.Join(" > ", processedPathAsList);
to get rid of another valiable.